qemu/tests/virtio_net_dpdk.py

```diff
@@ -206,7 +208,7 @@ def result(recode, dst):
     process.system_output("numactl --hardware")
     process.system_output("numactl --show")
     _pin_vm_threads(params.get("numa_node"))
-    error_context.context("Prepare env of vm/generator host", logging.info)
+    error_context.context("Prepare env of vm/generator host", test.log.info)
```

Comment:
For L77 and L104,  miss `test.log.info` for `error_context.context`.   Not this patch's scope,  just thinking fix them together.

Reply:
The same as the above.